WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginListFragment.java

```diff
@@ -0,0 +1,306 @@
+package org.wordpress.android.ui.plugins;
+
+import android.app.Fragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.ColorRes;
+import android.support.annotation.DrawableRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.RatingBar;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.fluxc.Dispatcher;
+import org.wordpress.android.fluxc.model.SiteModel;
+import org.wordpress.android.fluxc.model.plugin.SitePluginModel;
+import org.wordpress.android.fluxc.model.plugin.WPOrgPluginModel;
+import org.wordpress.android.fluxc.store.PluginStore;
+import org.wordpress.android.ui.ActivityLauncher;
+import org.wordpress.android.ui.plugins.PluginBrowserActivity.PluginListType;
+import org.wordpress.android.util.HtmlUtils;
+import org.wordpress.android.widgets.DividerItemDecoration;
+import org.wordpress.android.widgets.WPNetworkImageView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+public class PluginListFragment extends Fragment {
+
+    public static final String TAG = PluginListFragment.class.getName();
+    private static final String ARG_LIST_TYPE = "list_type";
+
+    public interface PluginListFragmentListener {
+        List<?> onListFragmentRequestPlugins(@NonNull PluginListFragment fragment);
+        void onListFragmentLoadMore(@NonNull PluginListFragment fragment);
+    }
+
+    private RecyclerView mRecycler;
+    private PluginListType mListType;
+
+    private boolean mCanLoadMore = true;
+    private boolean mIsLoadingMore;
```

Comment:
I believe we need to retain both `mCanLoadMore` and `mIsLoadingMore` in config changes.